### ISSUE TITLE
On the wait screen, show who has submitted so we know who's left

### ISSUE THREAD

**Issue by melindamorang** [melindamorang/blowyourfaceoff#66]
Just an enhancement.

**Comment by melindamorang** [comment]
To do this, we would have to ping more than just the round status table, thereby negating some of the efficiency enhancements we made by adding a round status table in the first place.  We would have to update the schema or otherwise request more information from the database. I'm not sure I want to do this.  I don't know if this enhancement is worth it, particularly now that we have a timer that will just move the slow people along eventually anyway.

**Comment by sgriffin231** [comment]
Didn't you already do this?

**Comment by melindamorang** [comment]
Nope. This is about the wait screen between rounds, not the lobby.
